File: src/AulaPratica02/Exercicio02/dados/Endereco.java
package AulaPratica02.Exercicio02.dados;

public class Endereco {
    private String rua;
    private String numero;
    private String bairro;
    private String cidade;
    private String estado;
    private String cep;

    public void setRua(String rua) {
        this.rua = rua;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }

    public void setBairro(String bairro) {
        this.bairro = bairro;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public void setCep(String cep) {
        this.cep = cep;
    }

    @Override
    public String toString() {
        return "* Estado: " + this.estado + "\n* Cidade: " + this.cidade + "\n* Bairro: " + this.bairro + "\n* Rua: " + this.rua + "\n* Numero: " + this.numero + "\n* CEP: " + this.cep;
    }

}

File: src/Lista01/Questao05/Endereco.py
import string

class Endereco:

	def __init__(self, rua, numero, bairro, cidade, estado, cep):
		self.__rua = rua
		self.__numero = numero
		self.__bairro = bairro
		self.__cidade = cidade
		self.__estado = estado
		self.__cep = cep

	def getRua(self):
		return self.__rua
	
	def setRua(self, rua: string):
		self.__rua = rua

	def getNumero(self):
		return self.__numero
	
	def setNumero(self, numero: int):
		self.__numero = numero

	def getBairro(self):
		return self.__bairro

	def setBairro(self, bairro : string):
		self.__bairro = bairro

	def getCidade(self):
		return self.__cidade

	def setCidade(self, cidade: string):
		self.__cidade = cidade

	def getEstado(self):
		return self.__estado

	def setEstado(self, estado : string):
		self.__estado = estado

	def getCep(self):
		return self.__cep
	
	def setCep(self, cep :string):
		self.__cep = cep

	def __str__(self):
		return 'Rua: {}, Numero: {}, Bairro: {}, Cidade: {}, Estado: {}, Cep: {}'.format(self.__rua, self.__numero, self.__bairro, self.__cidade, self.__estado, self.__cep)